internal/validate/validate.go

```diff
@@ -93,7 +93,7 @@ func ObjectID(s string) error {
 	return nil
 }
 
-var ErrMissingCredentials = errors.New("missing credentials")
+var ErrMissingCredentials = errors.New("this actions requires authentication")
```

Comment:
this action or these actions
```suggestion
var ErrMissingCredentials = errors.New("this action requires authentication")
```